ethereum/.env.prod.aurora

```diff
@@ -5,3 +5,5 @@ MIGRATIONS_NETWORK=aurora
 WORMHOLE_BRIDGE_ADDRESS=0xa321448d90d4e5b0A732867c18eA198e75CAC48E
 PYTH_TO_WORMHOLE_CHAIN_ID=0x1
 PYTH_TO_WORMHOLE_EMITTER=0x6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25
+
+VALID_TIME_PERIOD_SECONDS=15
```

Comment:
Can we set the initial time periods to err on the high side? I think some of these thresholds are likely too aggressive -- would suggest doubling all of them as a baseline, and using 60 seconds as a minimum. (Note that your calculation also doesn't account for clock drift between the chains, which could be substantial)

I suggest this because we are currently not in a position to measure how many failures occur because the threshold is too aggressive. Without that measurement, we could end up in a situation where Pyth is extremely flaky on a target chain, and we wouldn't know it. We can optimize these thresholds later once our measurement infrastructure is in place.

Reply:
These are my initial estimates based on the block time of the given chain. But we can measure it once we deploy attester in Pythnet. 

I shared my opinion about clock drift in the above thread too. I don't think it be an issue.

Reply:
I applied your suggestion.